experiments/nf_evaluation/datasets.py

```diff
@@ -0,0 +1,77 @@
+import pandas as pd
+
+from datasetsforecast.m4 import M4
+from datasetsforecast.m3 import M3
+from datasetsforecast.long_horizon import LongHorizon
+
+
+def get_dataset(name):
```

Comment:
the name of the method is not correct. You are returning only the dataset but also other parameters